### PR TITLE
Add more icons to design package

### PR DESCRIPTION
**Pull request by rudream**
### Purpose

Add icons to our design package to match the icons [I added for sales center webapp](https://github.com/gravitational/cloud/pull/661), this way when we migrate this design package to the cloud repo, it doesn't overwrite and delete those icons.

### Icons Added:

image
image
image
image
image
image

### CODE REVIEW

**Review by kimlisa — APPROVED** [review]

**Comment by kimlisa** [comment]
please wait for @alex-kovoy approval

**Review by alex-kovoy — APPROVED** [review]

**Pull request merged**
